src/ray/raylet/worker_pool.cc

```diff
@@ -1436,7 +1449,7 @@ void WorkerPool::PopWorker(const TaskSpecification &task_spec,
         }
         return callback(worker, status, runtime_env_setup_error_message);
       });
-  PopWorker(std::move(pop_worker_request));
+  PopWorker(std::move(pop_worker_request), std::move(serialized_allocated_instances));
```

Comment:
The `serialized_allocated_instances` is const &，so it does not need to `std::move` here.